vignettes/setup/setup.R

```diff
@@ -19,12 +19,17 @@ knit_print.grViz <- function(x, ...) {
     knitr::asis_output()
 }
 
-# If input loads dm...
+# If input loads dm or tidyverse, we load it here to omit warnings
```

Comment:
I learned today that we can also load {conflicted} to avoid those warnings, at least in reprexes: https://github.com/mgirlich/tibblify/issues/97. Does that also apply to vignettes?

No need to take action here, perhaps a new issue?

Reply:
Created a new [issue](https://github.com/cynkra/dm/issues/1172) to track this.